### PR TITLE
Add cucumber tests for "How It Works" link

### PR DESCRIPTION
**Pull request by pcardune**

### CODE REVIEW

File: dashboard/test/ui/step_definitions/steps.rb [hunk 1]
```diff
@@ -53,6 +53,10 @@ def color_for_status(status)
   wait_with_timeout.until { @browser.find_element(selection_criteria) }
 end
 
+When /^I go to the newly opened tab$/ do
+  @browser.switch_to.window(@browser.window_handles.last)
```
Comment (islemaster) [inline dashboard/test/ui/step_definitions/steps.rb:57]: This is slick :+1:

**Comment by islemaster** [comment]
I'm trying to verify this on SauceLabs and getting a lot of failures.  Here are the logs (which contain links to the SauceLabs playbacks).
- [iPhone](https://cucumber-logs.s3.amazonaws.com/pcardune-fix-how-it-works-link/iPhone_applab_embed_output.html?versionId=PhyEeSFxCGytlB0hmZXtHxEfqxpaWCVo)
- [iPad](https://cucumber-logs.s3.amazonaws.com/pcardune-fix-how-it-works-link/iPad_applab_embed_output.html?versionId=H.Lu6pNPItj0YugV7A9HWab2dTSz9ft5)
- [Safari](https://cucumber-logs.s3.amazonaws.com/pcardune-fix-how-it-works-link/SafariYosemite_applab_embed_output.html?versionId=yhD3U92OjVfwxwagd8XufXzlT7rlGUyg)
- [IE11](https://cucumber-logs.s3.amazonaws.com/pcardune-fix-how-it-works-link/IE11Win10_applab_embed_output.html?versionId=0wEj6q6SUdmmM0y48AqtVCZOe692ee4Q)
- [FirefoxLatest-2Yosemite](https://cucumber-logs.s3.amazonaws.com/pcardune-fix-how-it-works-link/FirefoxLatest-2Yosemite_applab_embed_output.html?versionId=TWsZgZHgHoMsRyl8N2YDHGKKitndmlZa)
- [Firefox45Win7](https://cucumber-logs.s3.amazonaws.com/pcardune-fix-how-it-works-link/Firefox45Win7_applab_embed_output.html?versionId=qO7Gk5lV0n9SbzZaQPRAQxdkej1EnAJe)
- [ChromeLatestWin7](https://cucumber-logs.s3.amazonaws.com/pcardune-fix-how-it-works-link/ChromeLatestWin7_applab_embed_output.html?versionId=ltsSmktC3qmb4BW9yyT5JZ67FiqBd6a1)
- [Chrome44XP](https://cucumber-logs.s3.amazonaws.com/pcardune-fix-how-it-works-link/Chrome44XP_applab_embed_output.html?versionId=njMMSEbYHIRJfZpGKtogs9cqKezLZeb8)

Most of them time out on "I wait to see Applab code mode."  From the visual logs, I'm not sure the iOS selenium driver even supports going to a new tab.  On the other hand, Chrome looks like it switches and loads everything just fine, but the "wait for" command doesn't pick up the thing it's looking for.

**Comment by pcardune** [comment]
hmm, perhaps we should just try to open the link in the parent frame instead of in a new window/tab.

**Comment by islemaster** [comment]
That seems reasonable, although if you think opening a new window/tab is a better experience I'd hate to move away from that just because it's harder to test.

I don't think we open new windows/tabs very often on our site though, so this may be the right choice.

**Comment by pcardune** [comment]
@poorvasingal do you have an opinion about whether or not click "How It Works" from inside an embed should open a new window/tab vs open in the containing frame?

**Comment by pcardune** [comment]
poorva sent me a message. one more vote for open in the same window.

**Pull request merged**